tests/e2e/config/jest-puppeteer.config.js

```diff
@@ -0,0 +1,10 @@
+const { useE2EJestPuppeteerConfig } = require( '@woocommerce/e2e-environment' );
+
+const puppeteerConfig = useE2EJestPuppeteerConfig( {
+	launch: {
+		slowMo: 100,
```

Comment:
Do we still need `slowMo`?

Reply:
Irritatingly, the `slowMo` value here is actually required. The `slowMo` basically sets a delay in between lines of execution while the E2E tests are running. Without the `slowMo` value in both headless and non-headless mode, the tests (particularly package tests) will fail periodically, because elements are not added to the DOM in time. In particular, regarding the package tests, a new package will be created and to confirm that it has been created correctly Jest will check the DOM for the new row containing the package before it has been added to the DOM and the test will fail. 

As an aside, this was particularly painful to debug and diagnose, because stepping through using the debugger would show no issues, but then the tests would fail from time to time when it came to running them at normal speed.  There may be room to decrease this number to make the tests run a bit quicker, but I found that keeping it 💯 hit the sweet spot, where the tests would consistently not fail randomly.

Reply:
Thanks for the explanation!